benchmark/nixlbench/src/runtime/etcd/etcd_rt.cpp

```
@@ -151,6 +151,19 @@ xferBenchEtcdRT::cleanupForExit() {
     }
 }
 
+bool
+xferBenchEtcdRT::checkKeepAlive() {
```

Cppcheck (2.20.0)

[style] The function 'checkKeepAlive' is never used.

(unusedFunction)